Source/diablo.cpp

```diff
@@ -14,6 +14,13 @@
 
 namespace devilution {
 
+#ifndef DEFAULT_WIDTH
+#define DEFAULT_WIDTH 640
+#endif
+#ifndef DEFAULT_HEIGHT
+#define DEFAULT_HEIGHT 480
+#endif

```

Comment:
Shouldn't these be in `diablo.h` instead?

Also, why the `#ifndef` around them? Can we remove?

Reply:
its only used in diablo.pp, some devices overwrite it because of there screen size and not having hardware capable of upscaling.